experimental/GITFans/GITFans.jl

```diff
@@ -416,16 +423,21 @@ function fan_traversal(orbit_list, q_cone, perm_actions)
 
         # note that we run also over elements added inside the loop
         current_cone_list = cones_from_bitlist(orbit_list, current_hash)
-        intersected_cone = Polymake.polytope.intersection(current_cone_list...)
-        facets = intersected_cone.FACETS
+        intersected_cone = current_cone_list[1]
+        for i in current_cone_list
+            intersected_cone = intersect(intersected_cone, i)
+        end
+        # TODO extend intersect to more than two cones
```

Comment:
I think thanks to @lkastner this could now be changed to this?
```suggestion
        intersected_cone = intersect(current_cone_list...)
```
BTW according to Codecov, this code is not executed as part of CI?

Reply:
Yeah, that was the intention. It turned out to be harder than expected, since instead of being a `Vector{Cone{T}}`, the variable `current_cone_list` only holds a `Vector{Any}`. Nevertheless, it works now.